steps/generate_analytical_dataset.py

```diff
@@ -58,6 +66,8 @@ def spark_process(collection):
     prefix = "${file_location}/" + collection.collection_name + '/' + collection.collection_name + ".parquet"
     tag_objects(prefix, tag_value=collection.tag_value)
     create_hive_on_published(parquet_location, collection.collection_name)
+    end_timer = time.perf_counter()
```

Comment:
I believe , We probably need another logger  fro across collections ?

Reply:
I have added this in now @UdayChokkam at the very end of the script